Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest-filter/ContestFilter.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useCallback, useMemo, useState } from 'react';
+
+import Heading, { HeadingType } from '../../guidelines/headings/Heading';
+import List, { Orientation } from '../../guidelines/lists/List';
+import { FilterType, IFilter } from '../../../common/contest-types';
+import ExpandButton from '../../guidelines/buttons/ExpandButton';
+import Button, { ButtonSize, ButtonType } from '../../guidelines/buttons/Button';
+import concatClassNames from '../../../utils/class-names';
+
+import styles from './ContestFilter.module.scss';
+
+
+interface IContestFilterTypeProps {
```

Comment:
Is it correct to specify 'Type' in the name of the interface? Isn't it better to be just 'IContestFilterProps' to be clear that the component is ContestFilter and the props for the component are a type of interface 'IContestFilterProps'

Reply:
Changed to IContestFilterProps.